doc/styleguide.md

```diff
@@ -0,0 +1,74 @@
+# Styleguide - Do's and Don't's
+
+Please adhere to this guide if you want to contribute any assets to the project.
+
+## Why you should follow this guide
+
+Game assets usually have a purpose beyond looking pretty. They are also visual aides to the player, helping them to achieve what they want. Players might admire your shiny graphic asset at first but in a tense multiplayer battle there will be no time for that. Therefore, your assets should not only be nicely drawn but also recognizable, functional and fitting for players' needs.
+
+Don't worry, you still have a lot of creative freedom!
+
+## Buildings
+
+### Civ architecture
+
+* Building sprites should use the architecture of their civilization.
+* Keep in mind that every architecture requires building sprites for Feudal, Castle and Imperial Age for some buildings.
+
+### Shape
+
+* Most buildings have unique shape (Examples to be done) that is consistent throughout civilization architectures. Try to follow that shape, when you create a building from scratch.
+* You don't have to follow the exact shape for every architecture. As a rule of thumb: The more a player frequents a building, the closer it should follow its unique shape. Examples are below.
+  * High frequency: Town center, barracks, archery range, stable
+  * Medium frequency: Market, mill, lumber/mining camp
+  * Low frequency: University, house
+* Monasteries and wonders have no fixed unique shape. Be creative with your models here.
+
+### Size
+
+* Building sprites should neither exceed their foundations nor be smaller than them. Otherwise players who use buildings to wall off their base will get confused whether there are holes in their defense.
+
+### Colors
+
+* Building sprites should have markings that can be used for the player colors. They shouldn't dominate the building but also shouldn't be too subtle.
+* Make sure that all 8 supported player colors will be visible on the markings. None of them should blend in with the building.
+
+## Units
+
+### Models
+
+* Unit models don't have to be historically accurate.
+* Models should be recognizable and not be too similar to other unit models.
+* Parts of the armor have to be reserved for the player colors. Make sure that all 8 supported colors work with your model.
+* Also make sure that the above rules are guaranteed **for every direction** the unit can face.
+
+### Animations
+
+* Attack animations should be clearly visible **for every direction** the unit can face.
+* You should provide the following animations for each unit:
+  * idle
+  * walking
+  * attacking
+  * dying
+  * decaying
+* Try to fit the length of an animation to the units attack speed.
+
+## Sounds
+
+* Sounds should not be longer than 3 seconds.
```

Comment:
Ideally we want sounds that are short and precise because they are more memorable that way and can easier be recognized by players. There're also a lot of actions that trigger sounds, so the chance of them blending into each other is higher the longer the sounds are. 3 seconds is already a generous limit in my opinion. An experienced player usually presses his mouse button with a rate of about 1click/s, hence sound designers should aim for 1 second lengths.

And yes, there are two (out of 1328) sounds in the original game that barely exceed the 3 second limit :smile: They play when a building is burning. Most of the sounds are around or below 1 second. I also think that exceptions should be allowed but I wanted to set some limit. Otherwise people will start to submit 10 second long audio files that we have to cut ourselves.